packages/sync/src/modules/WP_Super_Cache.php

```diff
@@ -16,7 +18,7 @@ public function __construct() {
 	);
 
 	static $wp_super_cache_callables = array(
-		'wp_super_cache_globals' => array( 'Jetpack_Sync_Module_WP_Super_Cache', 'get_wp_super_cache_globals' ),
+		'wp_super_cache_globals' => array( 'Automattic\\Jetpack\\Sync\\Modules\\WP_Super_Cache', 'get_wp_super_cache_globals' ),
```

Comment:
I wonder if we can use `__CLASS__` here instead 🤔 

Untested suggestion:
```suggestion
		'wp_super_cache_globals' => array( __CLASS__, 'get_wp_super_cache_globals' ),
```